src/Files.App/Views/Properties/MainPropertiesPage.xaml.cs

```diff
@@ -82,61 +68,56 @@ private async void Page_Loaded(object sender, RoutedEventArgs e)
 		{
 			AppSettings.ThemeModeChanged += AppSettings_ThemeModeChanged;
 
-			if (_usingWinUI)
+			if (FilePropertiesHelpers.IsWinUI3)
 			{
-				// WinUI3: Set rectangle for the Titlebar
+				// Set rectangle for the Titlebar
 				TitlebarArea.SizeChanged += (_, _) => DragZoneHelper.SetDragZones(Window, (int)TitlebarArea.ActualHeight);
-				DragZoneHelper.SetDragZones(Window, (int)TitlebarArea.ActualHeight);
-				AppWindow.Destroying += AppWindow_Destroying;
+				DragZoneHelper.SetDragZones(Window, (int)TitlebarArea.ActualHeight, 40);
+				Window.Closed += Window_Closed;
 
 				await App.Window.DispatcherQueue.EnqueueAsync(() => AppSettings.UpdateThemeElements.Execute(null));
 			}
 			else
 			{
-				_propertiesDialog = DependencyObjectHelpers.FindParent<ContentDialog>(this);
-				_propertiesDialog.Closed += PropertiesDialog_Closed;
+				_dialog = DependencyObjectHelpers.FindParent<ContentDialog>(this);
+				_dialog.Closed += PropertiesDialog_Closed;
 			}
 		}
 
 		private void MainPropertiesPage_SizeChanged(object sender, SizeChangedEventArgs e)
-		{
-			UpdateDialogLayout();
-		}
-
-		private void UpdateDialogLayout()
 		{
 			MainPropertiesWindowNavigationView.PaneDisplayMode =
 				ActualWidth <= 600 ? NavigationViewPaneDisplayMode.LeftCompact : NavigationViewPaneDisplayMode.Left;
 
 			if (ActualWidth <= 600)
-				foreach (var item in NavViewItems) item.IsCompact = true;
+				foreach (var item in NavigationViewItems) item.IsCompact = true;
 			else
-				foreach (var item in NavViewItems) item.IsCompact = false;
+				foreach (var item in NavigationViewItems) item.IsCompact = false;
+
+			DragZoneHelper.SetDragZones(App.Window, (int)TitlebarArea.ActualHeight, 40);
 		}
 
 		private async void AppSettings_ThemeModeChanged(object? sender, EventArgs e)
 		{
 			await DispatcherQueue.EnqueueAsync(() =>
 			{
-				((Frame)Parent).RequestedTheme = ThemeHelper.RootTheme;
-
-				if (!_usingWinUI)
+				if (!FilePropertiesHelpers.IsWinUI3)
 					return;
 
+				((Frame)Parent).RequestedTheme = ThemeHelper.RootTheme;
+
 				switch (ThemeHelper.RootTheme)
 				{
 					case ElementTheme.Default:
 						AppWindow.TitleBar.ButtonHoverBackgroundColor = (Color)Application.Current.Resources["SystemBaseLowColor"];
 						AppWindow.TitleBar.ButtonForegroundColor = (Color)Application.Current.Resources["SystemBaseHighColor"];
 						break;
-
 					case ElementTheme.Light:
-						AppWindow.TitleBar.ButtonHoverBackgroundColor = Color.FromArgb(51, 0, 0, 0);
+						AppWindow.TitleBar.ButtonHoverBackgroundColor = Color.FromArgb(0x33, 0, 0, 0);
 						AppWindow.TitleBar.ButtonForegroundColor = Colors.Black;
 						break;
-
 					case ElementTheme.Dark:
-						AppWindow.TitleBar.ButtonHoverBackgroundColor = Color.FromArgb(51, 255, 255, 255);
+						AppWindow.TitleBar.ButtonHoverBackgroundColor = Color.FromArgb(0x33, 0xFF, 0xFF, 0xFF);
```

Comment:
Were are these colors from?

Reply:
Just converted it to hex codes to see more accurately.